src/k8s/cmd/k8s/k8s_x_snapd_config.go

```diff
@@ -64,11 +64,7 @@ func newXSnapdConfigCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 			ctx, cancel := context.WithTimeout(cmd.Context(), opts.timeout)
 			defer cancel()
 			if err := control.WaitUntilReady(ctx, func() (bool, error) {
-				_, partOfCluster, err := client.NodeStatus(cmd.Context())
-				if !partOfCluster {
-					cmd.PrintErrf("Node is not part of a cluster: %v\n", err)
-					env.Exit(1)
-				}
+				_, _, err := client.NodeStatus(cmd.Context())
```

Comment:
So, if `NodeStatus()` always returns `partofCluster=false` for any error why do we no longer need this exit early?  

i'm not sure i have the context from #633 to properly review this.  Maybe @mateoflorido

Reply:
basically you're saying any error from `NodeStatus` at all exits early-- and that's wrong?

Reply:
So, `WaitUntilReady` will retry if the bool return value is `false`. If something in the error return value is returned, then this exits immediately. 

`NodeStatus` second return value indicates if the node is part of a cluster or not. However, it will also return `false` for this argument if an error is returned (third return value). The current implementation would always exit with `Node is notpart of cluster` even if `NodeStatus` returned and error (which means the "is part of cluster" return value is not valid).

Reply:
AHH!  i didn't look down at lines 70-71.  Thank you